app/views/layouts/mailer.html.erb

```diff
@@ -18,10 +18,10 @@
         <div style="font-size:0.78em">
           <% if @unsubscribe %>
             <%= link_to "Unsubscribe", email_subscriptions_unsubscribe_url(ut: @unsubscribe), style: "color:#7d7d7d" %> |
-            <a href="https://dev.to/settings/notifications" style="color:#7d7d7d">Adjust your email settings</a>
+            <a href="<%= app_url("/settings/notifications") %>" style="color:#7d7d7d">Adjust your email settings</a>
```

Comment:
can we use the path here? `user_settings_path(:notifications)`:

```ruby
> Rails.application.routes.url_helpers.user_settings_path(:notifications)
=> "/settings/notifications"
```

Reply:
oooh nice one! will add it